lib/rules/jsx-newline.js

```diff
@@ -0,0 +1,63 @@
+/**
+ * @fileoverview Enforce a new line after jsx elements and expressions.
+ * @author Johnny Zabala
+ */
+
+'use strict';
+
+const arrayIncludes = require('array-includes');
+const docsUrl = require('../util/docsUrl');
+
+// ------------------------------------------------------------------------------
+// Rule Definition
+// ------------------------------------------------------------------------------
+
+module.exports = {
+  meta: {
+    docs: {
+      description: 'Enforce a new line after jsx elements and expressions',
+      category: 'Stylistic Issues',
+      recommended: false,
+      url: docsUrl('jsx-newline')
+    },
+    fixable: 'code'
+  },
+  create(context) {
+    const jsxElementParents = new Set();
+    const sourceCode = context.getSourceCode();
+    return {
+      'Program:exit'() {
+        jsxElementParents.forEach((parent) => {
+          parent.children.forEach((element, index, elements) => {
+            if (arrayIncludes(['JSXElement', 'JSXExpressionContainer'], element.type)) {
```

Comment:
```suggestion
            if (element.type === 'JSXElement' || element.type === 'JSXExpressionContainer') {
```
no need to make an array for just 2 items :-)

Reply:
Solved!